demo/src/screens/componentScreens/MaskedInputScreen.tsx

```diff
@@ -85,11 +97,24 @@ export default class MaskedInputScreen extends Component {
           </Text>
           <MaskedInput
             migrate
-            ref={r => (this.priceInput = r)}
+            ref={this.priceInput}
             renderMaskedText={this.renderPrice}
-            formatter={value => value?.replace(/\D/g, '')}
+            formatter={(value: string) => value?.replace(/\D/g, '')}
             keyboardType={'numeric'}
           />
+
+          <Text text70 marginT-s5 marginB-s4>
+            PIN Code
+          </Text>
+          <MaskedInput
+            migrate
+            maxLength={4}
+            ref={this.priceInput}
```

Comment:
You should create a dedicated `ref`, currently only one of the two users (price \ pin) are deleted.

Reply:
Fixed as well (: